featuretools/feature_base/feature_base.py

```diff
@@ -141,35 +149,46 @@ def get_dependencies(self, deep=False, ignored=None, copy=True):
 
 
         """
-        deps = []
+        hash_key = hash(f"{self.get_name()}{self.dataframe_name}{deep}{ignored}")
+        if cached_dependencies := self.cache.get(CacheType.DEPENDENCY, hash_key):
```

Comment:
A walrus in the wild - don't see many of those in our codebase!

Reply:
ikr?  If cleans things up rather nicely, i think

Reply:
Never knew about these till now. I love it!